### ISSUE TITLE
get_facet_mask is broken

### ISSUE THREAD

**Issue by tkarna** [thetisproject/thetis#177]
`utility.get_facet_mask` appears to be broken, breaking the 3D model. Below is a minimal example:

```python
from firedrake import *
mesh2d = UnitSquareMesh(10, 10)
mesh = ExtrudedMesh(mesh2d, 10, 0.1)
V = FunctionSpace(mesh, 'DG', 1)
from thetis import get_facet_mask
get_facet_mask(V, 'geometric', 'top')
# returns [0, 1, 2, 3, 4, 5]
get_facet_mask(V, 'geometric', 'bottom')
# also returns [0, 1, 2, 3, 4, 5]
# expected: the top/bottom nodes of a prism, e.g. [0, 1, 2] and [3, 4, 5]
```

**Comment by tkarna** [comment]
It looks like this is a consequence of Firedrake now defaulting to spectral elements (firedrakeproject/firedrake#1537). The following works in the minimal example:
```
helem = FiniteElement('DG', mesh2d.ufl_cell(), 1, variant="equispaced")
velem = FiniteElement('DG', interval, 1, variant="equispaced")
elem = TensorProductElement(helem, velem)
V = FunctionSpace(mesh, elem)
```

**Comment by tkarna** [comment]
Fixed in #178.
